src/python/tests/core/test_scheduler.py

```diff
@@ -308,27 +364,34 @@ class TestScheduler:
 
         await asyncio.sleep(5)
         tasks = await scheduler.get_running_tasks()
-        await scheduler.cancel_task(tasks[0].task_id)
+
+        await scheduler.cancel_task(tasks[0].task_id)  # Cancel a running task
 
         await self.stop_scheduler(scheduler)
 
     @pytest.mark.asyncio
     async def test_get_schedule(self):
+        """Schedule gets retrieved
+        :assert:
+            Schedule is retrieved by id """
         scheduler = Scheduler()
 
         await scheduler.populate_test_data()
         await scheduler.start()
 
+        # Declare schedule
```

Comment:
These kinds of comments that separate sections of a 'def' are useful for readability